src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/repository/AttendanceRepositoryTest.kt

```diff
@@ -91,9 +91,9 @@ class AttendanceRepositoryTest {
 
   @Test
   fun `should throw error on missing fields`() {
-    Assertions.assertThrows( DataIntegrityViolationException::class.java) {
+    assertThatThrownBy {
       attendanceRepository.save(Attendance.builder().build())
-    }
+    }.isInstanceOf(ConstraintViolationException::class.java)
```

Comment:
This was switched out in previous PR incorrectly.  The reason why it stopped working was that the spring boot no longer included the hibernate validator stuff, but the hmpps gradle plugin now includes it automatically so things have started working again and validation is happening at java layer before hitting database